tests/functional/generate_pages.py

```diff
@@ -515,7 +517,7 @@ def process_schema(in_schema, out_dir, spec_file, require_path='..'):
     else:
         if os.path.isdir(args.SCHEMA):
             for root, dirs, files in os.walk(args.SCHEMA):
-                for file in [os.path.join(root, file) for file in files if file.startswith(args.test_schema_prefix)]:
+                for file in [os.path.join(root, file) for file in files if file.startswith(args.test_schema_prefix) or file.startswith(args.census_schema_prefix)]:
```

Comment:
We could probably just make this generate pages for all schemas? It's fast and doesn't get uploaded into the repo anymore. 

Not a big deal though.

Reply:
Sure, I think all the schemas match these filters anyway. Updated.